charmcraft.yaml

```diff
@@ -4,17 +4,6 @@ parts:
     source: .
     plugin: reactive
     build-snaps: [charm/2.x/stable]
```

Comment:
This might need updating to the latest `charm` snap.

Reply:
still install fail

Reply:
it appears that after split builds, many dependency can be removed